controllers/database/postgresql.go

```diff
@@ -1,36 +1,73 @@
 package database
 
 import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/go-logr/logr"
 	goharborv1 "github.com/goharbor/harbor-cluster-operator/api/v1"
+	"github.com/goharbor/harbor-cluster-operator/controllers/k8s"
 	"github.com/goharbor/harbor-cluster-operator/lcm"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
 )
 
 type PostgreSQLReconciler struct {
 	HarborCluster *goharborv1.HarborCluster
+	Ctx           context.Context
+	Client        k8s.Client
+	Recorder      record.EventRecorder
+	Log           logr.Logger
+	DClient       k8s.DClient
+	Scheme        *runtime.Scheme
+	ExpectCR      *unstructured.Unstructured
+	ActualCR      *unstructured.Unstructured
+	Labels        map[string]string
+	Name          string
+	Namespace     string
+	CRStatus      *lcm.CRStatus
+	Connect       *Connect
+	Properties    *lcm.Properties
```

Comment:
There is CRStatus, why you need Properties? It is redundancy？

Reply:
`Properties` will be used in `DeployComponentSecret`.